app/views/spots/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   <% if @spot.persisted? %>
     <%= f.input :status, as: :radio_buttons, collection: [["draft", "Draft"], ["published", "Published"]], label_method: :second, value_method: :first %>
   <% end %>
-  <%= f.input :name, input_html: { autocomplete: "off" } %>
+  <%= f.input :name, label: 'Spot Name', input_html: { autocomplete: "off" } %>
```

Comment:
Please use double quotes.

Also, no need to capitalize "Name" – as to better reflect the style of the rest of the labels.

🚀

Reply:
I've made these changes myself already, by the way! Just good for you to know for a next time. :-)